plugins/quickbars.md

```diff
@@ -10,19 +10,20 @@ The Quick Toolbar plugin (`quickbars`) enables new user interface components to
 
 It replaces the capabilities of the `inlite` theme in TinyMCE 4 or earlier.
 
-The Quick Toolbar plugin enables two new context toolbars:
+The Quick Toolbar plugin enables three new context toolbars:
 
-* _Quick Selection_ - shown when text is selected for quick access to formatting commands such as bold, italic and link.
+* _Quick Selection_ - shown when text is selected for quick access to formatting buttons such as bold, italic and link.
 * _Quick Insert_ - shown when a new line is created for the quick insertion of objects such as tables and images.
+* _Quick Image_ - shown when an image or figure is selected for quick access to image formatting buttons such as alignment.
 
-It also enables three new toolbar controls:
+It also enables three new toolbar buttons:
 
-* _Quick Link_ - an inline experience for creating and editing links without a dialog
-* _Quick Image_ - immediately prompts a user to select a local image to upload
-* _Quick Table_ - immediately inserts a 2x2 table without prompting the user to select the number of rows and columns
+* _Quick Link_ - an inline dialog for creating and editing links without a dialog.
```

Comment:
I don't think dialog is right here? I'm pretty sure this will use a context form, so there's no dialog that opens.